pkg/controller/quota/quotaforestmanager/qm_lib_backend_with_quotasubt_mgr.go

```diff
@@ -523,7 +523,7 @@ func (qm *QuotaManager) Fits(aw *arbv1.AppWrapper, awResDemands *clusterstateapi
 
 	_, err = qm.quotaManagerBackend.AddConsumer(consumerInfo)
 	if err != nil {
-		return false, nil, err.Error()
+		klog.Errorf("[Fits] Consumer '%s' already exists.", consumerInfo.GetID())
```

Comment:
@metalcycling I think this change is not necessary, and it decreases the ease of code comprehension.

It is not necessary because the call [allocResponse, err := qm.quotaManagerBackend.AllocateForest(QuotaManagerForestName, consumerID)](https://github.com/z103cb/multi-cluster-app-dispatcher/blob/24813bd30d760d88741a22225dbb389442054420/pkg/controller/quota/quotaforestmanager/qm_lib_backend_with_quotasubt_mgr.go#L531) will have the same effect as current implementation (see [if forestController.IsConsumerAllocated(consumerID) {](https://github.com/z103cb/multi-cluster-app-dispatcher/blob/24813bd30d760d88741a22225dbb389442054420/pkg/quotaplugins/quota-forest/quota-manager/quota/quotamanager.go#L436-L439))

This change is not helping the code comprehension as:
- logging an error and continuing is a bad practice. 
- if you do need to handle the error, it should probably be logged using the  _warning_ level.

Overall this change should be reverted as:
- it only makes code quality worse
- it doesn't fix the underlying issue (allocating consumers when they are already allocated)